Dockerfile

```diff
@@ -1,32 +1,44 @@
-FROM alpine:edge as builder
+FROM alpine:latest as builder
 
 COPY . /source/
-COPY bin/config.toml /etc/sozu/sozu.toml
 
-RUN apk add --no-cache --virtual .build-dependencies \
+RUN apk update && apk add --no-cache --virtual .build-dependencies \
   cargo \
   build-base \
   file \
   libgcc \
   musl-dev \
   rust
-RUN apk add --no-cache openssl-dev llvm-libunwind pkgconfig
-ENV SOZU_CONFIG /etc/sozu/sozu.toml
-ENV SOZU_PID_FILE_PATH /run/sozu/sozu.pid
-WORKDIR /source/ctl
-RUN cargo build --release
+RUN apk add --no-cache openssl-dev \
+  llvm-libunwind \
+  pkgconfig
+
 WORKDIR /source/bin
-RUN cargo build --release
 
+RUN cargo build --release --features use-openssl
+
+
+
+FROM alpine:latest as bin
+
+COPY bin/config.toml /etc/sozu/config.toml
```

Comment:
please use another file than `bin/config.toml`. A small one that activates HTTP, HTTPS and TCP proxies, with default values for buffer size, etc, should be ok. We should encourage people to use their own configuration file

Reply:
I created a empty configuration file at `os-build/docker/config.toml`. We should find a way to manage all this configs file in `os-build`.